### PR TITLE
cmd/bosun: Add rule links to alert status panels

### PR DESCRIPTION
**Pull request by kylebrandt**

### CODE REVIEW

File: cmd/bosun/web/static/js/state.ts [hunk 1]
```diff
@@ -111,6 +111,18 @@ bosunApp.factory('status', ['$http', '$q', function($http: ng.IHttpService, $q:
...
+						'alert=' + encodeURIComponent(btoa(v.AlertDef)) +
+						'&template=' + encodeURIComponent(btoa(v.TemplateDef)) +
+						'&fromDate=' + encodeURIComponent(v.last.Time.format("YYYY-MM-DD")) +
+						'&fromTime=' + encodeURIComponent(v.last.Time.format("HH:mm"));
+					var groups: string[] = [];
+					angular.forEach(v.Group, (v, k) => {
+						groups.push(k + "=" + v);
+					});
+					if (groups.length > 0) {
+						v.RuleUrl = v.RuleUrl + '&template_group=' + encodeURIComponent(groups.join());
```
Comment (mjibson) [inline cmd/bosun/web/static/js/state.ts]: Join needs an argument, '&'

Comment (mjibson) [inline cmd/bosun/web/static/js/state.ts]: v.RuleUrl += is shorter than v.RuleUrl = v.RuleUrl + ...

Comment (mjibson) [inline cmd/bosun/web/static/js/state.ts]: Does join need an argument here? `','`, I think.

Comment (kylebrandt) [inline cmd/bosun/web/static/js/state.ts]: Mozilla says the default is a comma, which is what I wanted: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

**Pull request merged**